src/Dfc.CourseDirectory.WebV2/AddressSearch/GetAddressAddressSearchService.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Flurl;
+using Newtonsoft.Json;
+
+namespace Dfc.CourseDirectory.WebV2.AddressSearch
+{
+    public class GetAddressAddressSearchService : IAddressSearchService
+    {
+        private const string IdDelimiter = "::";
+
+        private readonly HttpClient _httpClient;
+        private readonly Options _options;
+
+        public GetAddressAddressSearchService(HttpClient httpClient, Options options)
```

Comment:
If `Options` is specific to `GetAddress` then I think its name should indicate that

Reply:
Agreed